client.go

```diff
@@ -246,24 +264,28 @@ func (client *NativeClient) Shell(args ...string) error {
 		ssh.ECHO: 1,
 	}
 
-	fd := os.Stdin.Fd()
+	fd := int(os.Stdin.Fd())
 
-	if term.IsTerminal(fd) {
```

Comment:
It may have been added into the go lib since this was written, but if not term is preferred. Thanks